src/server/models/users.model.js

```diff
@@ -70,7 +70,12 @@ let userSchema = new Schema({
   education: [educationSchema],
   experience: [experienceSchema],
   work: [experienceSchema],
-  career: [careerSchema],
+  career: {
+    area: String,
```

Comment:
If we only support Engineering and Design for now, it'd be good to make these as enum rather than string (same goes for role below). Doesn't need to happen (probably can't..) in this PR, but either leave a TODO or keep that in mind :)